File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.py
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        da = [True for i in range(26)]
        
        for s in sentence:
            index = ord(s) - ord('a')
            if da[index]:
                da[index] = False
                
        for i in da:
            if i:
                return False
            
        return True

File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.go
func checkIfPangram(sentence string) bool {
    hashMap := make(map[byte]int)
    for i:= 'a'; i <= 'z'; i++ {
        hashMap[[]byte(string(i))[0]] = 0
    }
    for i := range sentence {
        hashMap[sentence[i]]++
    }
    for _, v := range hashMap {
        if v == 0 {
            return false
        }
    }
    return true
}